features/google/src/model/entities/place-visit.js

```diff
@@ -1,13 +1,33 @@
 import TimelineEvent from "./timeline-event";
 
 export default class PlaceVisit extends TimelineEvent {
-    constructor({ timestamp, locationName }) {
+    constructor({ timestamp, locationName, duration }) {
```

Comment:
Don't think we should necessarily pass the duration here and extract the `startTimestampMs`/`endTimestampMs` from it. The importer uses `extractTimestampFromDuration` as the timestamp can be either `startTimestamp` or `startTimestampMs`. Might be easier if the importer would extract and pass the endTimestamp alongside `timestamp`